README.md

```diff
@@ -9,7 +9,7 @@ approach than other available options like
 of curated collections of packages linked to a specific version/commit
 of Idris2, which are guaranteed to properly work together (otherwise,
 that's a bug in the package collection). This is similar to what
-*stack* for Haskell does: It avoids dependency hell by design.
+_stack_ for Haskell does: It avoids dependency hell by design.
```

Comment:
Could you please explain the motivation for this change?

Reply:
The motivation is to have a reference/documentation of how to upgrade my Idris 2 compiler to the updated version. I always need to search github issues and discord to find the correct command.